caffe2/CMakeLists.txt

```diff
@@ -244,9 +808,9 @@ target_include_directories(caffe2 SYSTEM PRIVATE "${Caffe2_DEPENDENCY_INCLUDE}")
 # Set standard properties on the target
 torch_set_target_props(caffe2)
 
-if (NOT MSVC)
-  target_compile_options(caffe2 INTERFACE "$<$<COMPILE_LANGUAGE:CXX>:-std=c++11>")
-endif()
+#if (NOT MSVC)
```

Comment:
why is this off?

Reply:
Seems redundant with [this line](https://github.com/pytorch/pytorch/blob/320f47f66529fd85164542105a0119ef2d0f5c6d/caffe2/CMakeLists.txt#L637):

    set_property(TARGET caffe2 PROPERTY CXX_STANDARD 11)